src/System.Collections.Concurrent/dir.props

```diff
@@ -2,6 +2,6 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.14.0</AssemblyVersion>
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
```

Comment:
Yes, I believe so. cc: @ericstj @stephentoub Should we log an issue to track reverting that change and bring the assembly version down again given that api changes were only for netcoreapp1.1?